Notify/Notify/Notify/ViewModels/NotificationsPageViewModel.cs

```diff
@@ -229,7 +232,34 @@ private void SetField<T>(ref T field, T value, [CallerMemberName] string propert
             field = value;
             OnPropertyChanged(propertyName);
         }
-        
+
+        private async void onAcceptNotificationButtonClicked(string notificationID)
+        {
+            string messageTitle = "Notification Acceptance";
+            string messageBody;
+            bool isConfirmed;
+            Notification notification;
+
+            isConfirmed = await App.Current.MainPage.DisplayAlert(messageTitle,
+                "Are you sure you want to accept this notification?",
+                "Yes", "No");
+
+            if (isConfirmed)
+            {
+                notification = new Notification(notificationID);
+
+                AzureHttpClient.Instance.UpdateNotificationsStatus(
+                    new List<Notification> { notification },
+                    Constants.NOTIFICATION_STATUS_ACTIVE
+                );
+
+                OnNotificationsRefreshClicked();
+
+                messageBody = $"Notification accepted successfully";
```

Comment:
Why not declare this line in the beginning like the 'messageTitle' or to declare the 'messageTitle' here too?

Reply:
Fixed.